examples/advanced_usage/send_embed.py

```diff
@@ -1,10 +1,15 @@
+import httpx
 from atproto import Client, models
 
+# image of a condescending cat stare
+DEFAULT_IMAGE_URL = 'https://cdn.bsky.app/img/feed_fullsize/plain/did:plc:xbtmt2zjwlrfegqvch7fboei/bafkreiae4rcokecag5qlpvg6l3otzulqn3hllnazmz2ezyqfl6xzpy5noe@jpeg'
```

Comment:
pls use image from this account https://bsky.app/profile/test.marshal.dev

Reply:
@MarshalX done!

![image](https://cdn.bsky.app/img/avatar/plain/did:plc:kvwvcn5iqfooopmyzvb4qzba/bafkreicwqcugdgubtawr6xv6jjifoju67eigwx2xgz4zs73nkzzn36oucy@jpeg)